nukkit.jar.conf

```
@@ -1,6 +1,6 @@
 [config]
 name = NukkitX
```

While you are in here can you rename to `Cloudburst Nukkit`
```suggestion
name = Cloudburst Nukkit
```